src/lib/data.json

```diff
@@ -0,0 +1,219 @@
+{
+    "supportedYears": [
+      {
+        "value": "2018",
+        "label": "2018"
+      },
+      {
+        "value": "2019",
+        "label": "2019"
+      },
+      {
+        "value": "2020",
+        "label": "2020"
+      },
+      {
+        "value": "2021",
+        "label": "2021"
+      },
+      {
+        "value": "2022",
+        "label": "2022"
+      }
+    ],
+    "personType": [
+      {
+        "value": "single",
+        "label": "Single"
+      },
+      {
+        "value": "separate",
+        "label": "Separate"
+      },
+      {
+        "value": "together",
+        "label": "Together"
+      },
+      {
+        "value": "head",
+        "label": "Head"
+      }
+    ],
+    "yearToTax": {
```

Comment:
Oh wait, one more thing, I just realized "yearToTax" is unecessary now as well. I think we originally had it to make it so our dictionary didn't get too big, but we really don't need it now that everything's in a json now, so can you remove "yearToTax" as well?

If you do this, you'll also need to remove yearToTax in lines 13 and 17 of App.svelte too.